wakuv2/waku.go

```diff
@@ -999,6 +1021,22 @@ func (w *Waku) broadcast() {
 				}
 			}
 
+			// Wraps the publish function with a call to the telemetry client
+			if w.statusTelemetryClient != nil {
+				sendFn := fn
+				fn = func(env *protocol.Envelope, logger *zap.Logger) error {
+					err := sendFn(env, logger)
+					if err == nil {
+						w.statusTelemetryClient.PushSentEnvelope(env, publishMethod)
+					}
+					// else {
+					// TODO: send error from Relay or LightPush to Telemetry
+					// w.statusTelemetryClient.PushError(err)
+					// }
```

Comment:
Will this be implemented as part of the PR? if not, then it's probably worth to create an issue to not forget

Reply:
Issue created: https://github.com/status-im/status-go/issues/5260